icons/buttondown.svg

```diff
@@ -0,0 +1,3 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>
+		Buttondown
+	</title><path d="M9.512.047c-.043.023-.297.055-.563.07C7.75.184 6.355.473 5.531.824c-.09.043-.207.09-.258.114A8.2 8.2 0 0 0 2.691 2.69 8.2 8.2 0 0 0 .918 5.32c-.047.1-.098.23-.121.28-.067.143-.258.76-.344 1.08a13.6 13.6 0 0 0-.336 2.265c-.015.266-.047.524-.07.57-.067.118-.067 4.852 0 4.97.023.046.055.304.07.57.067 1.195.356 2.59.707 3.414.043.09.09.207.114.258a8.2 8.2 0 0 0 1.753 2.582 8.2 8.2 0 0 0 2.63 1.773c.1.047.23.098.28.121.141.067.758.258 1.079.344a13.6 13.6 0 0 0 2.265.336c.266.015.524.047.57.07.118.067 4.852.067 4.97 0 .046-.023.304-.055.57-.07 1.195-.067 2.59-.356 3.414-.707.09-.043.207-.09.258-.113a8.2 8.2 0 0 0 2.582-1.754 8.2 8.2 0 0 0 1.773-2.63c.047-.1.098-.23.121-.28.067-.141.258-.758.344-1.079a13.6 13.6 0 0 0 .336-2.265c.015-.266.047-.524.07-.57.067-.118.067-4.852 0-4.97-.023-.046-.055-.304-.07-.57-.067-1.195-.356-2.59-.707-3.414-.043-.09-.09-.207-.113-.258a8.2 8.2 0 0 0-1.754-2.582 8.2 8.2 0 0 0-2.63-1.773c-.1-.047-.23-.098-.28-.121C18.257.73 17.64.539 17.32.453a13.6 13.6 0 0 0-2.265-.336c-.266-.015-.524-.047-.57-.07-.118-.063-4.872-.063-4.973 0m7.511 5.043C18 5.34 18.66 6 18.91 6.977c.067.261.074.835.074 5.023s-.007 4.762-.074 5.023c-.25.977-.91 1.637-1.887 1.887-.261.067-.832.074-4.96.074-4.04 0-4.704-.007-4.958-.07-.972-.242-1.601-.86-1.898-1.852-.066-.23-.074-.753-.074-5.062s.008-4.832.074-5.062q.438-1.472 1.871-1.848c.215-.055 1.031-.067 4.953-.07 4.168-.004 4.73.003 4.992.07M6.965 10.309l-.793.793.805.796c1.796 1.793 2.449 2.22 4.015 2.641.32.086 1.79.086 2.11 0 1.574-.426 2.187-.824 3.96-2.57l.868-.856-.797-.8c-.438-.438-.817-.797-.844-.797-.023 0-.453.406-.96.906-.5.496-.97.945-1.036.996a4.5 4.5 0 0 1-1.281.766 3.5 3.5 0 0 1-.54.125 1.8 1.8 0 0 1-.753.02c-1.028-.161-1.617-.548-3.164-2.106-.383-.387-.723-.707-.75-.707s-.403.355-.84.793"/></svg>
```

Comment:
Make sure that this is all on one line please - feel free to run the linter locally too! `npm install && npm run lint`.